### ISSUE TITLE
RTL languages support?

### ISSUE THREAD

**Issue by peylight** [RainLoop/rainloop-webmail#2073]
Can rainloop suppport RTL languages?

**Comment by akissz** [comment]
Yes it should support this. Only takes some css and a little php. Surprised its not already there. Hope someone else replies.
